README.md

```diff
@@ -200,6 +222,18 @@ Arguments:
 Note:
 - this link will be encrypted to the group you're linking to (i.e. link will have `recps: [groupId]`)
 
+### `ssb.tribes.link.createSubgroupLink({ group, subgroup }, cb)`
+
+Creates a message of tyoe `link/group-subgroup` which links a group to a subgroup
+
+Arguments:
+
+- `group` *GroupId* - the id of the parent private group
+- `subgroup` *GroupId* - the id of the subgroup you're linking to `group`
+- `cb` - *Function* - call with signature `(err, link)` where `link` is the link message
+Note:
+- this link will be encrypted to the group you're linking to (i.e. link will have `recps: [groupId]`)
```

Comment:
here are two groups. I would suggest saying "this will be encrypted to the _parent_ group"